Limitations of Test Messages (#3887)
braze-inc/braze-docs: _docs/_developer_guide/platform_wide/sending_test_messages.md

}' https://rest.iad-01.braze.com/messages/send
```

> ## Limititations of Test Messages

Reviewer: ```suggestion
## Limitations of test messages
```

> There are a few situations where Test Messages don't have complete feature parity with launching a campaign/canvas to a real set of users. In these instances to validate this behaviour you should launch the campaign/canvas to a limited set of test users.

Reviewer: ```suggestion
There are a few situations where test messages don't have complete feature parity with launching a campaign/canvas to a real set of users. In these instances, to validate this behavior you should launch the campaign/canvas to a limited set of test users.
```

> There are a few situations where test messages don't have complete feature parity with launching a campaign or canvas to a real set of users. In these instances, to validate this behavior you should launch the campaign or canvas to a limited set of test users.

Reviewer: ```suggestion
There are a few situations where test messages don't have complete feature parity with launching a campaign or Canvas to a real set of users. In these instances, to validate this behavior you should launch the campaign or Canvas to a limited set of test users.
```

> - Viewing the Braze [preference centre](16) from Test Messages will cause the submit button to be greayed out

Reviewer: ```suggestion
- Viewing the Braze [preference center][16] from **Test Messages** will cause the submit button to be grayed out
```

> - The list-unsubscribe header is not included in emails sent by the Test Message functionality

Reviewer: ```suggestion
- The list-unsubscribe header is not included in emails sent by the test message functionality
```

> - For in-app messages and content cards the target user must have a push token for the target device

Reviewer: ```suggestion
- For in-app messages and Content Cards, the target user must have a push token for the target device
```

[14]: https://dashboard-01.braze.com/app_settings/api_settings/
[15]: https://dashboard-01.braze.com/users/user_search/user-search/
[66]: {{site.baseurl}}/api/endpoints/messaging/send_messages/post_send_messages/

> [16]: https://www.braze.com/docs/user_guide/message_building_by_channel/email/managing_user_subscriptions/#subscription-groups

Reviewer: ```suggestion
[16]: {{site.baseurl}}/user_guide/message_building_by_channel/email/managing_user_subscriptions/#subscription-groups
```
